### PR TITLE
docs updates for mindsdb schema and projects

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
## Description

docs updates for mindsdb schema and projects

Fixes #issue_number

## Type of change

- [ ] 📄 This change is a documentation update

### CODE REVIEW

File: docs/sql/project.mdx [hunk 1]
```diff
@@ -3,303 +3,61 @@ title: MindsDB Projects
...
+CREATE <OBJECT> project_name.object_name
+...;
 ```
 
 <Note>
-   **Cannot Drop a Project**
-
-   Please note that if your project stores at least one object, it cannot be removed. In this case, you should first drop all the jobs and tables, such as models and views, belonging to this project, and then, you can remove the project.
-   Please see the [Example](#example) section for details.
+Please note that if you use the same name for a model and a view stored in one project, then MindsDB adds `_view` to the view name.
```
Comment (ea-rus) [inline docs/sql/project.mdx:45]: do we have a such feature (adding _view to name) ? I don't know about it

File: docs/sql/project.mdx [hunk 2]
```diff
@@ -3,303 +3,61 @@ title: MindsDB Projects
...
 </Note>
 
-## Working with `MODELS`
-
-### Creating `MODELS`
+### Drop a Project
 
-Here is how you create a model within the project:
+Use the beow command to remove a project.
```
Comment (ea-rus) [inline docs/sql/project.mdx:50]: typo in `below`

File: docs/sql/project.mdx (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/sql/project.mdx:45]: I was informed about it a while back, maybe it's obsolete now.
Let me remove it.

File: docs/mindsdb_sql/sql/api/manage-models-versions.mdx [hunk 3]
```diff
@@ -27,11 +27,11 @@ WHERE name = 'home_rentals';
 On execution, we get:
```
[the remark this answers is not in the document]
Comment (martyna-mindsdb) [reply docs/mindsdb_sql/sql/api/manage-models-versions.mdx:27]: Thank you. Replaced it with DESCRIBE MODEL and SHOW MODELS where relevant.

**Review by ea-rus — APPROVED** [review]

**Pull request merged**